src/components/Sum.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a, b }) => (
+  <p>
+    Sum of
+    {` ${a.toString()} `}
+    and
+    {` ${b.toString()} `}
+    is
+    {` ${(a + b).toString()} `}
+  </p>
```

Comment:
Ось в такому форматі відредагуй
```suggestion
  <p>
    {
     `text` 
     }
  </p>
```